package.json

```diff
@@ -23,6 +23,7 @@
   "devDependencies": {
     "chai": "^3.2.0",
     "coveralls": "^2.11.2",
+    "eslint": "^1.3.1",
```

Comment:
Can peasant take care of this `eslint` too? That way, we don't have to add too many dependencies. I assume that is what peasant is for.